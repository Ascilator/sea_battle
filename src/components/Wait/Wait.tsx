import React from 'react';

import { StyledWait } from './styles';

const Wait = () => <StyledWait>Wait!</StyledWait>;

export { Wait };
