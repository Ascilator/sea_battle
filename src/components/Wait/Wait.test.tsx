import renderer from 'react-test-renderer';
import { Wait } from './Wait';

test('Wait render', () => {
  const buttonGreen = renderer.create(<Wait />).toJSON();
  expect(buttonGreen).toMatchSnapshot();
});
