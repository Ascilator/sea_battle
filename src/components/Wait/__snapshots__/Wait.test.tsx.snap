// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wait render 1`] = `
<div
  className="sc-bdvvtL fa-DwsF"
>
  Wait!
</div>
`;
